.github/workflows/linux.yml

```diff
@@ -116,7 +116,7 @@ jobs:
       - name: CMake Build
         run: |
           source ${{ env.OV_INSTALL_DIR }}/setupvars.sh
-          cmake -DCMAKE_BUILD_TYPE=${{ matrix.build-type }} -S ${{ env.SRC_DIR}} -B ${{ env.BUILD_DIR }}
+          cmake -DOpenVINODeveloperPackage_DIR=${{ env.OV_INSTALL_DIR }}/developer_package/cmake -DCMAKE_BUILD_TYPE=${{ matrix.build-type }} -S ${{ env.SRC_DIR}} -B ${{ env.BUILD_DIR }}
```

Comment:
The comment in `CMakeLists.txt` says
> \# Find OpenVINODeveloperPackage first to compile with SDL flags

Is that the reason for the chanage?

Reply:
Yes, that is a prerequisite for starting to use GHA artifacts for distribution.